Makefile

```diff
@@ -78,7 +78,7 @@ binary: build
 build-local:
 	@rm -rf build/bundles/${MAJOR_VERSION}/binary
 	GO111MODULE=on CGO_ENABLED=1 go build ${TAGS_OPT} \
-		-ldflags "-B 0x$(shell head -c20 /dev/urandom|od -An -tx1|tr -d ' \n') -X main.Version=${MAJOR_VERSION}(${GIT_VERSION}) -X ${PROJECT_NAME}/pkg/types.IstioVersion=${ISTIO_VERSION}" \
+		-ldflags "-B 0x$(shell head -c20 /dev/urandom|od -An -tx1|tr -d ' \n') -X main.Version=${MAJOR_VERSION}(${GIT_VERSION}) -X ${PROJECT_NAME}/pkg/admin/server.Version=${MAJOR_VERSION}(${GIT_VERSION}) -X ${PROJECT_NAME}/pkg/types.IstioVersion=${ISTIO_VERSION}" \
```

Comment:
maybe it's better to reuse the existing main.Version, to avoid such similar code?

Reply:
Also, it's this line is tooo long, we should split it with `\`.

Reply:
> maybe it's better to reuse the existing main.Version, to avoid such similar code?
Then main.Version should not be under main.

Reply:
> Then main.Version should not be under main.

that sounds reasonable.

Reply:
sure